### DISCUSSION TITLE
Building from latest git clone doesn't generate the openCV DLLs in Windows

### DISCUSSION THREAD

**Discussion by amanabat** [emgucv/emgucv#804] [Q&A]
I would like to start off by saying I followed the instructions on building/pulling latest source (as of writing it's 4.6.0) for [Windows](https://emgu.com/wiki/index.php/Download_And_Installation#Building_from_Git) running the '_**Build_Binary_x86-64_doc.bat**_'
I've built a previous version before 4.3.0 and with that build I get the openCV binaries (opencv_aruco430.dll, opencv_bgsegm430, etc.) in the "**_emgu\libs\x64_**" directory and the static libraries in "**_emgu\b\bin\x64\Debug_**" folder. I don't see this same behavior with the latest source (4.6.0), So I was wondering if I'm just missing an extra parameter in the bat file or do I need to manually set the projects inside the emgucv.sln by hand to generate *.dlls? Any help would be greatly appreciated. Thanks.

**Comment by emgucv** [comment]
It is documented in the release note:
https://www.emgu.com/wiki/index.php/Version_History#Emgu.CV-4.6.0

> Moved compiled windows binary to "runtimes\win-x64\native" instead of "x64", "runtimes\win-x86\native" instead of "x86".
> Moved UWP binary output folder to "runtimes\win10-{arch}\native".
> Moved Mac OS binary from "libs\macos" to "libs\runtimes\osx\native".

**Reply by amanabat** [reply]
Thank you so much for the clarification. I will check the release notes as well next time. 
The only other thing I would like to ask is the openCV binaries built using the bat file specified above generated (opencv_perf_*.exe) instead of the opencv DLLs (please see screenshot below). Is there a switch I'm missing? 
![opencv_binaries](https://user-images.githubusercontent.com/49654054/212322507-3ef00733-0bd5-4131-ac45-df7d23402b9b.png)

**Reply by emgucv** [reply]
You can set the CMake flag to

> -DBUILD_PERF_TESTS:BOOL=OFF

https://github.com/emgucv/emgucv/blob/4.6.0/platforms/windows/Build_Binary_x86.bat#L343

to disabled building the performance. The open cv native binary is compiled into "cvextern.dll" and is available in your case.
